src/Amalgam/Amalgam.h

```diff
@@ -15,8 +16,16 @@
 
 extern "C"
 {
+	//status from LoadEntity
+	typedef struct
+	{
+		bool loaded;
+		char *message;
+		char *version;
+	} LoadEntityStatus_s;
```

Comment:
Moving to newer C style is fine, will update. The "_s" is because I also have the C++ one that doesn't have the postfix. So best way was to use the older style naming with postfix in C one. We don't have any namespaces for the C++ ones so there will be a collision unless you have another suggestion?

Reply:
Update C struct to not have postfix and moved C++ class into another class so no name collisions.